drivers/hmis/app/graphql/mutations/save_assessment.rb

```diff
@@ -2,48 +2,31 @@ module Mutations
   class SaveAssessment < BaseMutation
     description 'Create/Save assessment as work-in-progress'
 
-    argument :assessment_id, ID, 'Required if updating an existing assessment', required: false
-    argument :enrollment_id, ID, 'Required if saving a new assessment', required: false
-    argument :form_definition_id, ID, 'Required if saving a new assessment', required: false
-    argument :values, Types::JsonObject, 'Form state as JSON', required: true
-    date_string_argument :assessment_date, 'Date with format yyyy-mm-dd', required: false
+    argument :input, Types::HmisSchema::AssessmentInput, required: true
 
     field :assessment, Types::HmisSchema::Assessment, null: true
-    field :errors, [Types::HmisSchema::ValidationError], null: false
+    field :errors, [Types::HmisSchema::ValidationError], null: false, resolver: Resolvers::ValidationErrors
```

Comment:
Does it make sense to move this up to the BaseMutation?

Reply:
Yes it does, done